From: Wilton <wiljp@gmail.com>
Subject: [PATCH] meson: relax libdisplay-info dependency to >=0.2.0

Void Linux repositories provide libdisplay-info 0.2.0.
Relax the meson.build requirement from >=0.3.0 to >=0.2.0.

Signed-off-by: Wilton <wiljp@gmail.com>
---
diff --git a/meson.build b/meson.build
--- a/meson.build
+++ b/meson.build
@@ -121,1 +121,1 @@
-libdisplay_info_dep = dependency('libdisplay-info', version: '>=0.3.0')
+libdisplay_info_dep = dependency('libdisplay-info', version: '>=0.2.0')
